lib/plugins.js

```diff
@@ -109,7 +109,7 @@ exports.first = function(predicate) {
 rethrowUnlessFailedRequire = function(name, e) {
   var rethrow = true;
   if (e.code === 'MODULE_NOT_FOUND') {
-    var match = e.message.match(/^cannot find module '(.*)'$/i);
```

Comment:
What's an old vs a new error message look like?

node's errors have a `code` property that's far more reliable than using the message (in node versions that have the code). Could we prefer using that, and fall back to message-based heuristics?

Reply:
The `code` is already being used on the previous line. The regex is being used to confirm that the missing module is actually the plugin trying to be loaded, not some other dependency (that's my assumption, anyway).

**Node v12.4.1:**
```
Error: Cannot find module 'require("foo");'
Require stack:
- internal/preload
```

**Node v8.15.1:**
```
Error: Cannot find module 'require("foo");'
```